internal/sharederrors/error_help_links_test.go

```diff
@@ -0,0 +1,30 @@
+package sharederrors
+
+import (
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestHelpLinks(t *testing.T) {
+	t.Parallel()
+
+	c := http.Client{}
+	t.Cleanup(c.CloseIdleConnections)
+
+	for _, helpURL := range allErrorHelpUrls {
+		t.Run(helpURL, func(t *testing.T) {
+			t.Parallel()
+
+			require.Eventually(t, func() bool {
+				get, err := c.Get(helpURL)
+				require.NoError(t, err)
+				_ = get.Body.Close()
+				require.Equal(t, http.StatusOK, get.StatusCode)
+				return true
+			}, 30*time.Second, 100*time.Millisecond)
```

Comment:
Is `Eventually` spawning a goroutine? Like do we need to worry about nesting `require` calls here, or is `Eventually` just polling until the test completes?

Do we need an `Eventually` here? Or is the idea that it's going to help avoid network request issues?

Last thought: are we gonna get rate limited?

Reply:
Yeah, that makes sense. I'd also think that it'd be weird if you couldn't make `require` calls inside of `require.Eventually` - I wonder if they're doing special handling here.